src/yival/cli/init.py

```diff
@@ -28,21 +28,41 @@
 )
 from .utils import generate_experiment_config_yaml
 
+try:
+    from ..finetune.sft_trainer import SFTTrainer
+except ImportError:
+    print(
+        "[Warn] missing modules in trainers, ignore this warn if you don't want to finetune model in yival"
+    )
```

Comment:
A `try` block for `import` should be balanced. There must be a `SFTTrainer` as  backup. Or, inner code should be modified to makesure the import to be certainely success.

Reply:
Add backup trainer for SFTTrainer_backup

<img width="791" alt="image" src="https://github.com/YiVal/YiVal/assets/55043304/7c42c47a-8c22-4199-9169-22797f303350">

and it will raise error when train
<img width="757" alt="image" src="https://github.com/YiVal/YiVal/assets/55043304/cc5de64d-a393-4a27-a94b-86c7c3d6cc77">